crates/aiken-project/src/blueprint/schema.rs

```diff
@@ -297,18 +298,11 @@ impl Annotated<Schema> {
                             // make all types abide by this convention.
                             let data = match definitions.try_lookup(&generic).cloned() {
                                 Some(Annotated {
-                                    annotated: Schema::Data(Data::List(Items::Many(xs))),
+                                    annotated: Schema::Data(Data::Pair(left, right)),
```

Comment:
The schema here should likely be Pair, not Data

Reply:
Hmm I'll have to look into the conversion from Schema::Pair into a Schema of Data to handle the Map case